code/game/objects/items/weapons/storage/boxes.dm

```diff
@@ -546,6 +546,21 @@
 		new /obj/item/weapon/storage/pill_bottle( src )
 
 
+/obj/item/weapon/storage/box/spraybottles
+	name = "box of spray bottles"
+	desc = "It has pictures of spray bottles on its front."
+	New()

```

Comment:
Due to coding standards, can you redo this entire file to use the proper method of proc declaration?